src/base/QXmppStanza.h

```diff
@@ -135,7 +135,8 @@ class QXMPP_EXPORT QXmppStanza : public QXmppNonza
             SubscriptionRequired,       ///< The requester needs to subscribe first.
             UndefinedCondition,         ///< An undefined condition was hit.
             UnexpectedRequest,          ///< The request was unexpected.
-            PolicyViolation             ///< The entity has violated a local server policy. \since QXmpp 1.3
+            PolicyViolation,            ///< The entity has violated a local server policy. \since QXmpp 1.3
+            EmailConfirmationRequired,  ///< The server requires the entitiy to confirm an email message that the server sent to the entity. \since QXmpp 1.6
```

Comment:
that's no stanza error, right? i don't think i can accept that.

Reply:
The new QXmpp::StreamError is used now.